src/sa_web/static/js/views/place-list-view.js

```diff
@@ -210,74 +231,71 @@ var Shareabouts = Shareabouts || {};
       var sortFunction = this.sortBy + 'Sort';
 
       this.collection.comparator = this[sortFunction];
+      this.unrenderedItems.comparator = this[sortFunction];
       this.collection.sort();
+      this.unrenderedItems.sort();
```

Comment:
Oh, actually it might not matter--I think that since the default behavior of `.add()` is to sort models as they are added to a collection, the call to `this.unrenderedItems.add(this.collection.models)` on line 257 should be sorting `unrenderedItems`. But in that case the call to `this.collection.sort()` on line 235 might be unnecessary?